lib/rollbar/capistrano_tasks.rb

```diff
@@ -0,0 +1,131 @@
+require 'capistrano'
+require 'capistrano/version'
+require 'rollbar/deploy'
+
+module Rollbar
+  # Module containing the logic of Capistrano tasks for deploy tracking
+  module CapistranoTasks
+    class << self
+      def deploy_started(capistrano, logger, dry_run)
+        deploy_task(logger, :desc => 'Notifying Rollbar of deployment start') do
+          result = report_deploy_started(capistrano, dry_run)
+
+          info_request_response(logger, result)
+
+          capistrano.set(:rollbar_deploy_id, 123) if dry_run
+
+          skip_in_dry_run(logger, dry_run) do
+            if (deploy_id = result[:data][:deploy_id])
+              capistrano.set :rollbar_deploy_id, deploy_id
+            else
+              logger.error 'Unable to report deploy to Rollbar'
+            end
+          end
+        end
+      end
+
+      def deploy_succeeded(capistrano, logger, dry_run)
+        deploy_update(capistrano, logger, dry_run, :desc => 'Setting deployment status to `succeeded` in Rollbar') do
+          report_deploy_succeeded(capistrano, dry_run)
+        end
+      end
+
+      def deploy_failed(capistrano, logger, dry_run)
+        deploy_update(capistrano, logger, dry_run, :desc => 'Setting deployment status to `failed` in Rollbar') do
+          report_deploy_failed(capistrano, dry_run)
+        end
+      end
+
+      private
+
+      def deploy_task(logger, opts = {})
+        capistrano_300_warning(logger)
+        logger.info opts[:desc] if opts[:desc]
+        yield
+      end
+
+      def deploy_update(capistrano, logger, dry_run, opts = {})
+        deploy_task(logger, opts) do
+          depend_on_deploy_id(capistrano, logger) do
+            result = yield
+
+            info_request_response(logger, result)
+
+            skip_in_dry_run(logger, dry_run) do
+              if result[:response].is_a?(Net::HTTPSuccess)
+                logger.info 'Updated deploy status in Rollbar'
+              else
+                logger.error 'Unable to update deploy status in Rollbar'
+              end
+            end
+          end
+        end
+      end
+
+      def capistrano_300_warning(logger)
+        logger.warn("You need to upgrade capistrano to '>= 3.1' version in order to correctly report deploys to Rollbar. (On 3.0, the reported revision will be incorrect.)") if ::Capistrano::VERSION =~ /^3\.0/
```

Comment:
@ArturMoczulski Capistrano 2 (atleast version 2.12.0) doesn't have the `Capistrano::VERSION` constant.

Does the latest version of Rollbar not support Capistrano 2?